lib/consignment.js

```diff
@@ -275,8 +279,8 @@ var fetchProductsByConsignment  = function(args, connectionInfo, retryCounter) {
   } else {
     log.debug('retry # ' + retryCounter);
   }
-
-  var path = '/api/consignment_product';
+  var path = `/api/2.0/consignments/${args.consignmentId.value}/products`;
+  //var path = '/api/consignment_product';
```

Comment:
Remove this commented code before merging, or add the deprecation notice URL in the comment somewhere with reason @chronos25